### ISSUE TITLE
Method not found error on Gradient start

### ISSUE THREAD

**Issue by AndreLuettig** [losttech/Gradient#3]
Used version: 0.1.10-tech-preview5

When trying to start any calculations with Gradient, I get the following Exception:
The type initializer for "tensorflow.tf" threw an exception.

InnerException:
Method not found: "Void Python.Runtime.Runtime.set_PythonDLL(System.String)".

Stacktrace: 
   at Gradient.GradientSetup.PreloadPythonEmbeddingLibrary(PythonEnvironment pythonEnvironment)
   at Gradient.GradientSetup.Initialize()
   at Gradient.GradientSetup.EnsureInitialized()
   at tensorflow.tf..cctor()

It happens when using the default python environment (the one determined by GradientSetup - is working, since I get the following message: This trial version of Gradient will expire in 24 days. Please, download new version from NuGet.) and when manually choosing different Environments (was trying Anaconda Environment 3.6, 3.7 and Visual Studio Environment 3.7 with `GradientSetup.UsePythonEnvironment(usedEnv)`. The Environments were determined either by `PythonEnvironment.EnumerateEnvironments()` or `PythonEnvironment.EnumerateCondaEnvironments()`). Also tried setting the DynamicLibraryPath variable to a random value, to see if the assignment works (`{usedEnv}.DynamicLibraryPath = "";`). It works perfectly fine.

The code I wanted to run and where the error occured (one of the examples of the website): 
`var a = new dynamic[] { tf.constant(5.0, name: "a") };`

Can anyone help me with that?

**Comment by lostmsu** [comment]
Hi @AndreLuettig , try upgrading `Python.Runtime.UnofficialNetStandard` NuGet package to the latest preview: `2.4.0-a7`.

Also, I am not sure I understood the part of your comment regarding the environments. The warning that you see is not a good indicator of initialization success, as it is printed before `PreloadPythonEmbeddingLibrary`.

Additionally, I would check the process bitness. Currently, the host process must be 64-bit.

The last thing to check is to ensure you don't have `Python.Runtime` NuGet package installed in the same project, as the changes required by Gradient are not yet upstreamed to it. So you have to use `Python.Runtime.UnofficialNetStandard`

**Comment by lostmsu** [comment]
@AndreLuettig, if that does not resolve the problem, please, add the OS version and Visual Studio version that you are using.

**Comment by AndreLuettig** [comment]
Thank you! The one with `Python.Runtime.UnofficialNetStandard` did the trick. I also didn't have any `Python.Runtime` package installed, I just had a downloaded version of anaconda from the official website and another Python instance from Visual Studio Installer, so I did not know a `Python.Runtime.UnofficialNetStandard` was needed.
Thanks again for the quick reply and solution!

**Comment by lostmsu** [comment]
@AndreLuettig `Gradient` has `Python.Runtime.UnofficialNetStandard` in its dependencies. So it should have been downloaded and installed automatically.

**Comment by AndreLuettig** [comment]
Aaaah, I found my mistake. Since I couldn't find Gradient in NuGet in Visual Studio (I didn't check `Include prerelease`), I installed it using `Install-Package Gradient -Version 0.1.10-tech-preview5` in the Package Manager Console. And with that the `Python.Runtime.UnofficialNetStandard` was not installed. 
After checking `Include prerelease` and installing it from NuGet directly, the package was downloaded and installed as dependency!
